platform/darwin/test/MGLExpressionTests.mm

```diff
@@ -348,6 +355,12 @@ - (void)testStatisticalExpressionObject {
         XCTAssertEqualObjects([expression expressionValueWithObject:nil context:nil], @28);
         XCTAssertEqualObjects([NSExpression expressionWithMGLJSONObject:jsonExpression], expression);
     }
+    {
+        NSExpression *expression = [NSExpression expressionForFunction:@"sum:" arguments:@[@1, @2, @2, @3, @4, @7, @9]];
```

Comment:
I am not sure if this test case is valid or not. As far as I understand, we should not mix up `expressionWithFormat` and `expressionForFunction`. In this case, if we want to create NSExpression with `expressionForFunction` , the arguments shall be Expression as well, otherwise there is no difference between this two creation methods, and further more it will mess up the conversion of `expression.mgl_jsonExpressionObject`. That's why this test can not be passed. So I suggest that create the argument like following: 
```
 NSArray *arguments = @[MGLConstantExpression(@1), MGLConstantExpression(@2), MGLConstantExpression(@2), ...];
```
And there needs to be somewhere explain the differences of this two  Expression  creation ways.